secrets.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs');
 const util = require('util');
 
 const generateKeyPair = util.promisify(crypto.generateKeyPair);
-const padding = crypto.constants.RSA_PKCS1_PADDING;
+const padding = crypto.constants.RSA_PKCS1_OEAP_PADDING;
```

Comment:
I think you misspelled OAEP.

Reply:
Sheesh! That's embarrassing :) Nice catch, will fix. (This is why I prefer typescript.) Luckily, that means `padding` is undefined, and node-crypto defaults to `RSA_PKCS1_OAEP_PADDING`, and always has as far as I can tell. So no behavior difference.